mk_verificator/utils/__init__.py

```diff
@@ -38,7 +38,18 @@ def get_groups(config):
 
 
 def get_configuration(path_to_test):
+    """global conf is minor, but local conf is major."""
+
     test_folder = os.path.dirname(os.path.abspath(path_to_test))
     config_file = '/'.join([test_folder, "config.yaml"])
     config = yaml.load(open(config_file, 'r'))
-    return config
+
+    global_config_folder = '/'.join(os.path.dirname(
+        os.path.abspath(__file__)).split('/')[:-1])
```

Comment:
frankly speaking, better do not use split here. I would prefer kinda of:

just an example:
global_config_folder = os.path.join(current_file_path, '../../..')